linux/help/reference/km-package-get.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require_once('includes/template.php');
+require_once('includes/__docheader.php');
+
+$baselanguage = "language-bash";
+
+head([
+  'title' => "Command: km-package-get"
+]);
+
+?>
+
+<!-- TODO: Auto-generate from https://manpages.debian.org/unstable/keyman/km-package-get.1.en.html -->
```

Comment:
Do we have a task for these TODOs?

Reply:
I don't think we have a task. That comment was from @glasseyes